packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx

```diff
@@ -13,18 +13,18 @@ const propTypes = {
    * Callback function that is called when the group's selection state changes
    */
   onChange: PropTypes.func,
-  /**
-   * Indicates if the group should be selectable
-   */
-  isSelectable: PropTypes.bool,
   /**
    * CollapsibleMenuView.Items to be grouped together
    */
   children: PropTypes.node.isRequired,
+  /**
+   * A list of keys of the buttons that should be selected.
```

Comment:
A list of keys of the CollapsibleMenuView.Items that should be selected.

Reply:
Updated. https://github.com/cerner/terra-core/pull/1206/commits/c0f8bc677477b692dd3f318e73bf4a7937de70dd